XenAdmin/Dialogs/LicenseManager/LicenseStatus.cs

```diff
@@ -368,8 +365,7 @@ public string LicenseEntitlements
                     return Messages.LICENSE_NOT_ELIGIBLE_FOR_SUPPORT;
                 }
 
-                if ((PoolLicensingModel == LicensingModel.Creedence || PoolLicensingModel == LicensingModel.Clearwater)
-                    && CurrentState == HostState.Free)
+                if (CurrentState == HostState.Free)

```

Comment:
If the enum acquires one more member, then we'll have to revisit all its usages, but it should still be correct, because in general, if a pool is not licensed, then it is not eligible for support. And yes, I think the author (probably me 😄) meant `if (PoolLicensingModel != LicensingModel.PreClearwater)`